packages/client/src/components/Footer/Footer.style.css

```diff
@@ -0,0 +1,176 @@
+@import url('https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,300;0,400;0,500;0,600;0,700;0,800;1,300;1,400;1,500;1,600;1,700;1,800&display=swap');
+
+.footer-container {
+  font-family: 'Open Sans', sans-serif;
+  background-color: rgba(40, 48, 49);
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  width: 100vw;
+}
+
+.footer-content-section {
+  background-color: rgba(0, 0, 0, 0.3);
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 1220px;
+  height: 260px;
+}
+
+.footer-copyright-section {
+  background-color: rgba(23, 23, 23, 1);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 40px;
+  width: 100vw;
+}
+
+.footer-copyright-infosection {
+  font-size: 0.875;
+  color: rgba(255, 255, 255, 1);
+  width: 1220px;
+  text-align: right;
+}
+
+.placeholder-container {
+  width: 25rem;
+  height: 3.75rem;
+  border: 0.25rem solid rgba(188, 122, 0, 1);
+  padding: 0.06rem 0.93rem 0.06rem 1.56rem;
+}
+
+.placeholder-text {
+  color: rgba(245, 245, 245, 1);
+  font-weight: 600;
+  margin: auto;
+  width: 25rem;
+  height: 3.75rem;
+  display: flex;
+  align-items: center;
+}
+
+a {
```

Comment:
@ricardoaguiar  Can you clarify this statement a bit?

I imagine you mean that if we're styling by using a tag selector (like `a`) then we mean to style **all** anchor tags in our app like this, and in that case it makes sense to move those styles into global CSS.

But what if this styling is only for the links in the footer? Would we not then recommend changing the selector to a class name instead and keep it in the footer styles?

Reply:
@miltonhultgren I think he suggested it because our styling for a tag was just to remove the underline. And if there will be some case where it's necessary it can be overwritten.

Reply:
@miltonhultgren Sorry for the short statment, but @veronicaburduzhan captured the essence of it. 
And I agree that we should have a class name and use it to style the tag as needed.